LICENSE

```diff
@@ -1,3 +1,5 @@
+MIT License
```

Comment:
Added it just to be clear (we also have it mentioned in composer.json)

Reply:
Is there a specific need to add this since GitHub shows this anyway?
![2024-03-11_11-25-54](https://github.com/minkphp/webdriver-classic-driver/assets/1277526/4e063e23-8d09-468d-a5e9-d093c45a5d9a)

Reply:
@aik099 I'm not sure where GitHub gets that information from, I'm suspecting pattern matching the license text? 😅 

@stof this seems subjective. The heading [here is shown](https://choosealicense.com/licenses/mit/), but [not here](https://github.com/composer/composer/blob/main/LICENSE). Here are a few more: https://github.com/search?q=%22MIT+License%22+path%3ALICENSE&type=code&ref=advsearch